### ISSUE TITLE
TypeError: cannot set property 'tagger' of undefined in internet explorer 11

### ISSUE THREAD

**Issue by noobymatze** [elm-lang/virtual-dom#42]
The above error is thrown in the following line in internet explorer 11 on Windows 7: https://github.com/elm-lang/virtual-dom/blob/1.1.1/src/Native/VirtualDom.js#L1306

The actual problem is that the `elm_event_node_ref` is undefined. A SSCCE can be found here: https://gist.github.com/noobymatze/9b80606ccc69ad05e9b368b31532cc03

Typing any two letters into the rendered input element will result in the mentioned exception. Without opening the developer tools, nothing will happen. The typed text should be displayed right next to the input. Removing the `App.map` in the view and changing the type of `viewName` to `Html Msg` will resolve that.

I am not sure about anything below IE 11, but IE Edge, Chrome, Firefox and Safari 5.1.7 (on Windows) and Opera seem all to be working fine.

**Comment by evancz** [comment]
I am unable to reproduce this with IE11 on Windows 7 using the 0.18 alpha. What OS version are you on? If you use the [0.18 alpha](https://groups.google.com/forum/#!topic/elm-dev/LZslB1keLhE) and this slightly modified version of your program, can you reproduce?

```elm
module Main exposing (main)

import Html exposing (Html)
import Html.Attributes as A
import Html.Events as E
import Platform.Cmd


type alias Model =
  { name : String
  }


init : (Model, Cmd Msg)
init =
  ( Model ""
  , Cmd.none
  )


-- UPDATE

type Msg
  = UpdateName String
  | NoOp


update : Msg -> Model -> (Model, Cmd a)
update msg model =
  case msg of
    UpdateName name ->
      ({ model | name = name }, Cmd.none)

    NoOp ->
      (model, Cmd.none)


-- VIEW

view : Model -> Html Msg
view model =
  Html.div
    []
    [ Html.input [ A.type_ "text", A.value model.name, E.onInput UpdateName ] []
    , Html.map (always NoOp) <| viewName model.name
    ]


viewName : String -> Html Never
viewName =
  Html.text


main =
  Html.program
    { init = init
    , update = update
    , view = view
    , subscriptions = \_ -> Sub.none
    }
```

**Comment by noobymatze** [comment]
Works fine for me on 0.18 alpha. Sorry for the hassle. Should have tested it for myself on 0.18 first.

**Comment by evancz** [comment]
No worries. Weird that things were going wrong with 0.17.1 though. I'm gonna close given that it is "fixed" in 0.18. Please open a new thing if you see anything else like this!
